### PR TITLE
Create a new licensing non-fungible token standard request

### PR DESCRIPTION
**Pull request by d22shen**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by d22shen** [comment]
@MicahZoltu Hi Micah, sorry to trouble you. Is there any specific reason to close this PR? It's nice to know the root cause, so I can do edit and submit PR again. Thanks. Really appreciate it.

**Comment by MicahZoltu** [comment]
Sorry!  I must have accidentally closed it instead of something else I meant to close.

File: EIPS/eip-draft_lnft.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+---
+eip: <to be assigned>
```
Comment (MicahZoltu) [inline EIPS/eip-draft_lnft.md:2]:
```suggestion
eip: 4636
```
Please also rename the file to `EIPS/eip-4636.md`.

File: EIPS/eip-draft_lnft.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+title: Licensing Token Standard
+author: Bo Lin (@linvictor88), Dongsheng Shen (@d22shen)， Danyang Li (@muren102)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-12-22
+requires (*optional): <EIP number(s)>
```
Comment (MicahZoltu) [inline EIPS/eip-draft_lnft.md:10]:
```suggestion
```

**Comment by MicahZoltu** [comment]
Note: This won't get reviewed by anyone until you mark it as "Ready for Review".

**Comment by d22shen** [comment]
Ha, thanks a lot. Really appreciate it. Will do "Ready for Review" asap.

**Comment by d22shen** [comment]
This PR depends on https://github.com/ethereum/EIPs/pull/4635

**Pull request closed without merging**